content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -57,6 +66,15 @@ To delete a variable:
 1. Click the ellipses next to the variable you want to delete and select **Delete**.
 1. Click **Yes, delete variable** to confirm your action.
 
+## Loading Variables from Files
```

Comment:
I think we want this under "Workspace-Specific" right? This is the scope that they are (lmk if I'm wrong though).

```suggestion
### Loading Variables from Files
```

Reply:
OHHH okay gotcha! thank you for explaining this!